packages/cli/src/util/get-pagination-opts.ts

```diff
@@ -0,0 +1,23 @@
+interface Options {
```

Comment:
I think the function name is good, but `Options` type name is a little ambiguous. What kind of options are they?
```suggestion
interface PaginationOptions {
```